--- nixnote2-master.orig/NixNote2.pro	2016-02-03 17:37:47.000000000 +0300
+++ nixnote2-master/NixNote2.pro	2016-02-05 16:42:30.179662437 +0300
@@ -32,7 +32,7 @@ CONFIG(debug, debug|release) {
         MOC_DIR = build/debug
 }
 
-CONFIG += debug_and_release
+CONFIG += debug_and_release copy_dir_files
 
 TRANSLATIONS = \
     translations/nixnote2_cs_CZ.ts \
@@ -434,3 +434,13 @@ HEADERS  += nixnote.h \
 
 QMAKE_CXXFLAGS +=-g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security
 QMAKE_LFLAGS += -Wl,-Bsymbolic-functions -Wl,-z,relro
+
+RESOURCE_FILES = certs help images qss translations java
+
+target.path = $$[QT_INSTALL_PREFIX]/bin/
+resources.files += $$RESOURCE_FILES
+resources.path = $$[QT_INSTALL_PREFIX]/share/nixnote2/
+applications.files = nixnote2.desktop
+applications.path = $$[QT_INSTALL_PREFIX]/share/applications/
+INSTALLS += target resources applications
+
